pull-request.sh

```diff
@@ -70,15 +74,12 @@ create_pull_request() {
     else
         # Post the pull request
         DATA="{\"title\":${TITLE}, \"body\":${BODY}, \"base\":${TARGET}, \"head\":${SOURCE}, \"draft\":${DRAFT}, \"maintainer_can_modify\":${MODIFY}}"
-        printf "curl --user ${GITHUB_ACTOR} -X POST --data ${DATA} ${PULLS_URL}\n"
-        RESPONSE=$(curl -sSL -H "${AUTH_HEADER}" -H "${HEADER}" --user "${GITHUB_ACTOR}" -X POST --data "${DATA}" ${PULLS_URL})
-        RETVAL=$?
-        printf "Pull request return code: ${RETVAL}\n"
-
-        # if we were successful to open, add assignees and reviewers
-        if [[ "${RETVAL}" == "0" ]]; then
-
+        if ! RESPONSE=$(safe_curl -X POST --data "${DATA}" ${PULLS_URL}); then
+            curl_die "Error ${?} creating PR: ${RESPONSE}"
+        else
+            echo "::group::github response"
```

Comment:
What exactly is a group?

Reply:
It converts a section into something that collapses. See e.g. https://github.com/chronotope/chrono-tz/runs/959040913?check_suite_focus=true

<img width="249" alt="image" src="https://user-images.githubusercontent.com/277161/89671247-a4dcd100-d8b0-11ea-999a-25e29e71adfc.png">

and the arrow can be clicked on to expand:

<img width="232" alt="image" src="https://user-images.githubusercontent.com/277161/89671284-b6be7400-d8b0-11ea-8172-fa7235ef1fa5.png">

Reply:
Oh that's neat! Thanks you taught me something new, I'll definitely use that in the future for large unruly json output :)